src/components/PupilViews/PupilLessonOverview/PupilLessonOverview.view.tsx

```diff
@@ -1,67 +1,201 @@
+                "space-between-none",
+              ]}
+              $borderColor="bg-decorative1-main"
+              $pb={["inner-padding-l", "inner-padding-none"]}
+              $ph={["inner-padding-s", "inner-padding-none"]}
+              $bb={["border-solid-l", "border-solid-none", "border-solid-none"]}
             >
-              <OakFlex
-                $gap={"space-between-ssx"}
-                $justifyContent={"space-between"}
-              >
-                <OakSecondaryButton
-                  onClick={() => {
-                    updateCurrentSection(s);
-                  }}
+              {isValidIconName(subjectIconName) && (
+                <OakBox
+                  $mb={[
+                    "space-between-none",
+                    "space-between-none",
+                    "space-between-m",
+                  ]}
                 >
-                  {s}
-                </OakSecondaryButton>
-                <OakBox $width={"all-spacing-5"}>
-                  {completedSections.includes(s) && "✅"}
-                  {sectionResults[s] &&
-                    `${sectionResults[s]?.grade}/${sectionResults[s]?.numQuestions}`}
+                  <OakSubjectIcon
+                    iconName={subjectIconName}
+                    alt=""
+                    fill="bg-decorative1-main"
+                  />
                 </OakBox>
-              </OakFlex>
-            </OakLI>
-          ))}
-      </OakUL>
-      <OakPrimaryButton onClick={proceedToNextSection}>
-        Proceed to next section
-      </OakPrimaryButton>
-    </OakFlex>
+              )}
+              <OakBox>
+                <OakBox $mb="space-between-s" $display={["none", "block"]}>
+                  <OakBulletList
+                    listItems={[yearTitle, subjectTitle].filter(isString)}
+                    $color="text-subdued"
+                  />
+                </OakBox>
+                <OakHeading
+                  tag="h1"
+                  $font={["heading-7", "heading-5", "heading-3"]}
+                >
+                  {lessonTitle}
+                </OakHeading>
+              </OakBox>
+            </OakFlex>
+            {pupilLessonOutcome && (
+              <OakBox $display={["none", "block"]} $mt="space-between-xl">
+                <OakHeading tag="h2" $font="heading-7" $mb="space-between-s">
+                  Lesson outcome
+                </OakHeading>
+                <OakSpan $font="body-1">{pupilLessonOutcome}</OakSpan>
+              </OakBox>
+            )}
+          </OakGridArea>
+          <OakGridArea
+            $colStart={[1, 1, 7]}
+            $colSpan={[12, 12, 5]}
+            $ph={["inner-padding-s", "inner-padding-none"]}
+          >
+            <OakFlex $gap="space-between-s" $flexDirection="column">
+              <OakLessonNavItem
+                lessonSectionName="intro"
+                as="button"
+                onClick={() => updateCurrentSection("intro")}
```

Comment:
I can't remember where the discussion on whether it should be possible to jump past incomplete sections was left? Right now it's still possible to jump to any section.

Reply:
thats correct